### PR TITLE
Fix: Navigation Menu - Fixed active color issues in case of full page anchor links used in nav links.

### PR DESCRIPTION
**Pull request by swapnilde**
### Description
Navigation menu - Fixed active color issue in case of full page anchor links are used in nav links.

### Types of changes

Bug fix (non-breaking change which fixes an issue)

### How has this been tested?
1. Use custom links like `https://example.com/page#anchor`
2. Apply colors from available controls (Normal/Hover/Active)
3. Check on the frontend if all links get active color applied or the only current page link.

### Checklist:
- [x] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,9 +137,10 @@ This same applies when you are creating your Header/Footer using this plugin.
 5. Default widgets available with Elementor – Header, Footer & Blocks.
 
 ## Changelog ##
-### 1.5.10 ###
+### 1.5.9.1 ###
```
Comment (sushmak02) [inline README.md:140]: change version number here

File: inc/js/frontend.js [hunk 2]
```diff
@@ -67,6 +67,23 @@
 		var last_item = $( '.elementor-element-' + id + ' .hfe-nav-menu' ).data( 'last-item' );
 		var last_item_flyout = $( '.elementor-element-' + id + ' .hfe-flyout-wrapper' ).data( 'last-item' );
 
+		var menu_items_links = $( '.elementor-element-' + id + ' .hfe-nav-menu nav li a' );
+		menu_items_links.each(
+			function () {
+				if ($( this ).is( '[href*="#"]' )) {
```
Comment (sushmak02) [inline inc/js/frontend.js:73]: Store $( this ) in a variable instead of calling it repeatedly!

File: README.md (hunk 1 above)
Comment (swapnilde) [reply README.md]: done

File: inc/js/frontend.js (hunk 2 above)
Comment (swapnilde) [reply inc/js/frontend.js]: done

**Review by sushmak02 — APPROVED** [review]

**Pull request merged**